src/Components/Quote/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './index.css';
+
+
+export default class Quote extends Component {
+  render() {
+    return (
+        <p className="Quote">"{this.props.propQuote}" </p>
```

Comment:
I would use `this.props.children` instead of a regular prop here. This way we can use this component with more complex html structures if needed. Please take a look [here](https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891) for more an introduction.

Reply:
Seems like a good suggestion, I will update it. Thanks